### ISSUE TITLE
Alt text field missing / not required for some components

### ISSUE THREAD

**Issue by adamglenn** [wvuweb/wvu-ds-v3-bookshop-components#83]
For example the Hero - Split, Testimonials, and People pages (for profile pics).
